dataprofiler/profilers/graph_profiler.py

```diff
@@ -316,22 +346,20 @@ def _get_continuous_distribution(self, graph, continuous_attributes):
                 mean, variance, skew, kurtosis = distribution.stats(
                     best_fit_properties, moments="mvsk"
                 )
-                properties = list(best_fit_properties) + [
-                    mean,
-                    variance,
-                    skew,
-                    kurtosis,
-                ]
-
+                properties = {
```

Comment:
properties is being re-written as a dict() so it can be used as a cleaner data structure for test assertions, etc...